product/src/main/generated/transitive-dependencies-all.txt

```diff
@@ -2033,9 +2032,6 @@ org.apache.flink:flink-annotations
 org.apache.flink:flink-clients
 org.apache.flink:flink-connector-datagen
 org.apache.flink:flink-core
-org.apache.flink:flink-core-api
```

Comment:
Not sure why these flink artifacts get removed?

Reply:
I think this is expected. Before this change, the branch was aligned to 4.10.0 community which uses Flink 1.20.0:

https://github.com/apache/camel/blob/camel-4.10.0/parent/pom.xml#L154

But the midstream is aligned with 1.19.1:

https://github.com/jboss-fuse/camel/blob/camel-4.10.0-branch/parent/pom.xml#L154